internal/service/credentials/resource_credential_issuance_rule.go

```diff
@@ -0,0 +1,842 @@
+		MarkdownDescription: statusdDescriptionFmt,
+		Description:         strings.ReplaceAll(statusdDescriptionFmt, "`", "\""),
+	}
+
+	filterDescriptionFmt := "Contains one and only one filter (`.groupIds`, `.populationIds`, or `.scim`) that selects the users to which the credential issuance rule applies."
+	filterDescription := framework.SchemaDescription{
+		MarkdownDescription: filterDescriptionFmt,
+		Description:         strings.ReplaceAll(filterDescriptionFmt, "`", "\""),
+	}
+
+	automationOptionPhraseFmt := "Can be `PERIODIC` or `ON_DEMAND`." // I'm following the documentation here.
+	automationIssueDescriptionFmt := fmt.Sprintf("The method the service uses to issue credentials with the credential issuance rule. %s", automationOptionPhraseFmt)
+	automationIssueDescription := framework.SchemaDescription{
+		MarkdownDescription: automationIssueDescriptionFmt,
+		Description:         strings.ReplaceAll(automationIssueDescriptionFmt, "`", "\""),
+	}
+
+	automationRevokeDescriptionFmt := fmt.Sprintf("The method the service uses to revoke credentials with the credential issuance rule. %s", automationOptionPhraseFmt)
+	automationRevokeDescription := framework.SchemaDescription{
+		MarkdownDescription: automationRevokeDescriptionFmt,
+		Description:         strings.ReplaceAll(automationRevokeDescriptionFmt, "`", "\""),
+	}
+
+	automationUpdateDescriptionFmt := fmt.Sprintf("The method the service uses to update credentials with the credential issuance rule. %s", automationOptionPhraseFmt)
+	automationUpdateDescription := framework.SchemaDescription{
+		MarkdownDescription: automationUpdateDescriptionFmt,
+		Description:         strings.ReplaceAll(automationUpdateDescriptionFmt, "`", "\""),
+	}
+
+	notificationMethodsDescriptionFmt := "Array of methods for notifying the user; can be `EMAIL`, `SMS`, or both."
+	notificationMethodsDescription := framework.SchemaDescription{
+		MarkdownDescription: notificationMethodsDescriptionFmt,
+		Description:         strings.ReplaceAll(notificationMethodsDescriptionFmt, "`", "\""),
+	}
+
+	notificationTemplateLocaleDescriptionFmt := "The ISO 2-character language code used for the notification; for example, `en`."
+	notificationTemplateLocaleDescription := framework.SchemaDescription{
+		MarkdownDescription: notificationTemplateLocaleDescriptionFmt,
+		Description:         strings.ReplaceAll(notificationTemplateLocaleDescriptionFmt, "`", "\""),
+	}
+
+	resp.Schema = schema.Schema{
+		// This description is used by the documentation generator and the language server.
+		Description: "Resource to create and manage PingOne Credentials credential issuance rules.",
+
+		Attributes: map[string]schema.Attribute{
+			"id": framework.Attr_ID(),
+
+			"environment_id": framework.Attr_LinkID(framework.SchemaDescription{
+				Description: "PingOne environment identifier (UUID) in which the credential issuance rule exists."},
+			),
+
+			"credential_type_id": framework.Attr_LinkIDWithValidators(framework.SchemaDescription{
```

Comment:
Can be `Attr_LinkID` when PingOne ID is the only validator

Reply:
resolved